docs/user-guide/plugins.md

```diff
@@ -11,3 +11,4 @@ Plugins are rules and sets of rules built by the community that support methodol
 -   [`stylelint-scss`](https://github.com/kristerkari/stylelint-scss): Enforce a wide variety of SCSS-syntax specific linting rules (plugin pack).
 -   [`stylelint-selector-bem-pattern`](https://github.com/davidtheclark/stylelint-selector-bem-pattern): Specify a BEM pattern for selectors (incorporates [postcss-bem-linter](https://github.com/postcss/postcss-bem-linter)).
 -   [`stylelint-suitcss`](https://github.com/suitcss/stylelint-suitcss) - A collection of stylelint plugins for SUIT CSS, including deprecated `:root` rules (plugin pack).
+-   [`stylelint-images`](https://github.com/auth0/stylelint-images) - Check your CSS images to improve the performance and avoid common mistakes.
```

Comment:
"Check your CSS images to improve the performance and avoid common mistake (plugin pack)."